chromium_src/components/content_settings/core/browser/content_settings_registry.cc

```diff
@@ -3,101 +3,101 @@
-      WhitelistedSchemes(), \
-      ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK), \
-      WebsiteSettingsInfo::SINGLE_ORIGIN_ONLY_SCOPE, \
-      WebsiteSettingsRegistry::DESKTOP | \
-          WebsiteSettingsRegistry::PLATFORM_ANDROID, \
-      ContentSettingsInfo::INHERIT_IN_INCOGNITO, \
-      ContentSettingsInfo::PERSISTENT, \
-      ContentSettingsInfo::EXCEPTIONS_ON_SECURE_ORIGINS_ONLY); \
-  \
-  /* Disable motion sensors by default (brave/brave-browser#4789)*/ \
-  content_settings_info_.erase(ContentSettingsType::SENSORS); \
-  website_settings_registry_->UnRegister(ContentSettingsType::SENSORS); \
-  Register( \
-      ContentSettingsType::SENSORS, "sensors", CONTENT_SETTING_BLOCK, \
-      WebsiteSettingsInfo::UNSYNCABLE, WhitelistedSchemes(), \
-      ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK), \
-      WebsiteSettingsInfo::SINGLE_ORIGIN_ONLY_SCOPE, \
-      WebsiteSettingsRegistry::DESKTOP | \
-          WebsiteSettingsRegistry::PLATFORM_ANDROID, \
-      ContentSettingsInfo::INHERIT_IN_INCOGNITO, \
-      ContentSettingsInfo::PERSISTENT, \
-      ContentSettingsInfo::EXCEPTIONS_ON_SECURE_AND_INSECURE_ORIGINS); \
-  /* We need 62, 63, 64 for ContentSettingsTypes for Android tabs migration*/ \
-  /* We read only from these settings*/ \
-  Register( \
-      static_cast<ContentSettingsType>(62), "desktop-view", \
-      CONTENT_SETTING_BLOCK,  \
-      WebsiteSettingsInfo::UNSYNCABLE, \
-      WhitelistedSchemes(), \
-      ValidSettings(CONTENT_SETTING_BLOCK, CONTENT_SETTING_ALLOW), \
-      WebsiteSettingsInfo::REQUESTING_ORIGIN_AND_TOP_LEVEL_ORIGIN_SCOPE, \
-      WebsiteSettingsRegistry::PLATFORM_ANDROID, \
-      ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE, \
-      ContentSettingsInfo::PERSISTENT, \
-      ContentSettingsInfo::EXCEPTIONS_ON_SECURE_ORIGINS_ONLY); \
-  Register( \
-      static_cast<ContentSettingsType>(63), "play-video-in-background", \
-      CONTENT_SETTING_BLOCK, \
-      WebsiteSettingsInfo::UNSYNCABLE, WhitelistedSchemes(), \
-      ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK), \
-      WebsiteSettingsInfo::SINGLE_ORIGIN_ONLY_SCOPE, \
-      WebsiteSettingsRegistry::PLATFORM_ANDROID, \
-      ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE, \
-      ContentSettingsInfo::PERSISTENT, \
-      ContentSettingsInfo::EXCEPTIONS_ON_SECURE_ORIGINS_ONLY); \
-  Register( \
-      static_cast<ContentSettingsType>(64), "play-yt-video-in-browser", \
-      CONTENT_SETTING_ALLOW, \
-      WebsiteSettingsInfo::UNSYNCABLE, WhitelistedSchemes(), \
-      ValidSettings(CONTENT_SETTING_ALLOW, CONTENT_SETTING_BLOCK), \
-      WebsiteSettingsInfo::SINGLE_ORIGIN_ONLY_SCOPE, \
-      WebsiteSettingsRegistry::PLATFORM_ANDROID, \
-      ContentSettingsInfo::INHERIT_IF_LESS_PERMISSIVE, \
-      ContentSettingsInfo::PERSISTENT, \
-      ContentSettingsInfo::EXCEPTIONS_ON_SECURE_ORIGINS_ONLY); \
-}
+#define BRAVE_INIT                                                             \
```

Comment:
next time something like this comes up we should convert this to a method and pass in content_settings_info_, website_settings_registry_ and this so we don't have a giant multiline define